content/asp.net/getting-started/views.md

```diff
@@ -1,20 +1,121 @@
+
+## Using Razor View Engine Features
+
+When you employ these features in unison it is important to remember a few of the rules that govern their use by the view engine. Razor looks for direction on layouts and imports at each level in the folder structure. `Using` statements that bring in a namespace at the root level will be available in all subfolders. Likewise, tag helpers that are brought in at a subfolder level will not be available to other "sibling" folders unless you explicitly declare them there as well. Finally, `ViewImports` are additive and allow you to add or remove namespaces and tag helpers, but declaring imports at a subfolder level will not override or clear out previously imported components.
+
+In the project template we see a very basic set of statements in our `ViewImports.cshtml` document that expose the root project namespace and add in the Microsoft tag helpers library. This is the perfect place to add additional tag helpers that you may write yourself or pull in from other third-party libraries. Defining them here makes them available throughout all views in the project (due to the inheritance model used by Razor).
+
+``` csharp
+@using MvcBasics
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+```
+
+The `ViewStart` is also pretty straightforward in the template, but its use helps us to understand a little better what is going on. You can think of this code as the code that will be appended to the start of each class that is generated when our views are compiled. In this case, the layout that will be used for all views is assigned to the Layout property of the class. This can be overridden in views if needed. This would also be a good place to evaluate if a different layout should be used for the properties associated with the request, namely a user claim or perhaps the host name.
+
+``` csharp
+@{
+    Layout = "_Layout";
+    if (User.HasClaim("Role", "Admin"))
+    {
+        Layout = "_AdminLayout";
+    }
+}
+```
+
+Our `_Layout.cshtml` file is much more opinionated. It essentially creates the HTML page structure that our views will use including the containers and styles with which they are decorated. The template has to take on some dependencies on JavaScript and CSS libraries in order for this to be of any use to someone creating a web site, and you will see evidence of Bootstrap and jQuery here. We will leave full exploration of the HTML, CSS and JavaScript incorporated in this page to the reader, but there are a few important lines to point out in `_Layout.cshtml`. First is how the page head information is being set. 
```

Comment:
Seem to switch back and forth between "you" and "we". Let's stick with "you" throughout to refer to the reader.